modules/server/src/schema/Aggregations.ts

```diff
@@ -13,6 +18,7 @@ export let typeDefs = `
     key_as_string: String
     top_hits(_source:[String], size:Int): JSON
     filter_by_term(filter: JSON): JSON 
+    ${!enableDocumentHits ? 'belowThreshold: Boolean' : ''}
```

Comment:
guess this is another spot where `hideHits` would be semantically beneficial.
good comment for context! :+1: